[2025-04-21T16:59:44.240+0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T16:59:44.245+0800] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.fetch_weather manual__2025-04-21T07:00:36.233762+00:00 [queued]>
[2025-04-21T16:59:44.247+0800] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.fetch_weather manual__2025-04-21T07:00:36.233762+00:00 [queued]>
[2025-04-21T16:59:44.247+0800] {taskinstance.py:2867} INFO - Starting attempt 17 of 17
[2025-04-21T16:59:44.252+0800] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_weather> on 2025-04-21 07:00:36.233762+00:00
[2025-04-21T16:59:44.256+0800] {standard_task_runner.py:72} INFO - Started process 22613 to run task
[2025-04-21T16:59:44.260+0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_etl', 'fetch_weather', 'manual__2025-04-21T07:00:36.233762+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/var/folders/mz/6xjq0zld5fj5_cth7lsfpxlr0000gp/T/tmprdytl_r2']
[2025-04-21T16:59:44.261+0800] {standard_task_runner.py:105} INFO - Job 45: Subtask fetch_weather
[2025-04-21T17:01:24.614+0800] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2025-04-21T17:01:24.614+0800] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T17:01:24.617+0800] {process_utils.py:132} INFO - Sending 15 to group 22613. PIDs of all processes in the group: [22613]
[2025-04-21T17:01:24.617+0800] {process_utils.py:87} INFO - Sending the signal 15 to group 22613
[2025-04-21T17:01:24.621+0800] {process_utils.py:80} INFO - Process psutil.Process(pid=22613, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='16:59:44') (22613) terminated with exit code -15
